src/main/java/org/folio/marccat/resources/HeaderTypeAPI.java

```diff
@@ -1,25 +1,30 @@
 package org.folio.marccat.resources;
 
-import org.folio.marccat.config.constants.Global;
-import org.folio.marccat.resources.domain.HeadingTypeCollection;
-import org.springframework.web.bind.annotation.*;
-
 import static java.util.Optional.ofNullable;
 import static org.folio.marccat.config.constants.Global.BASE_URI;
 import static org.folio.marccat.integration.MarccatHelper.doGet;
 import static org.folio.marccat.resources.shared.MappingUtils.mapToHeading;
 
+import java.util.ArrayList;
+import java.util.List;
+
+import org.folio.marccat.config.constants.Global;
+import org.folio.marccat.resources.domain.HeadingType;
+import org.folio.marccat.resources.domain.HeadingTypeCollection;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
 @RestController
 @RequestMapping(value = BASE_URI, produces = "application/json")
 public class HeaderTypeAPI extends BaseResource {
 
-
   @GetMapping("/header-types")
-  public HeadingTypeCollection getHeadingTypes(
-    @RequestParam final String code,
-    @RequestParam final String lang,
-    @RequestHeader(Global.OKAPI_TENANT_HEADER_NAME) final String tenant,
-    @RequestHeader(Global.OKAPI_URL) String okapiUrl) {
+  public HeadingTypeCollection getHeadingTypes(@RequestParam final String code, @RequestParam final String lang,
```

Comment:
You can format with one annotation per line?

Reply:
I have changed it